contracts/test/v0.8/dev/VRFCoordinatorV2Mock.test.ts

```diff
@@ -262,7 +262,7 @@ describe('VRFCoordinatorV2Mock', () => {
       expect(receipt.events[0].args['success']).to.equal(true)
       assert(
         receipt.events[0].args['payment']
-          .sub(BigNumber.from('100119017000000000'))
+          .sub(BigNumber.from('100119403000000000'))
```

Comment:
If you have the time, some foundry tests on the mockfile could be nice. However, I don't think it's worth the effort ever adding typescript/hardhat tests, since we don't really plan on working with that software going forward.

Reply:
Sure, I'll try to add Foundry tests for VRFCoordinatorV2Mock. Shall I add it as a follow up PR to this?